### PR TITLE
Fast-track skopeo-1.5.1-1.fc35

### PR DESCRIPTION
**Pull request by cgwalters**
For https://github.com/coreos/enhancements/pull/7

### CODE REVIEW

File: manifest-lock.overrides.yaml [hunk 1]
```diff
@@ -8,4 +8,10 @@
 # in the `metadata.reason` key, though it's acceptable to omit a `reason`
 # for FCOS-specific packages (ignition, afterburn, etc.).
 
-packages: {}
+packages:
+  skopeo:
+    evr: skopeo-1.5.1-1.fc35
```
Comment (jlebon) [inline manifest-lock.overrides.yaml]: This needs to be

```suggestion
    evr: 1.5.1-1.fc35
```

Comment (cgwalters) [reply manifest-lock.overrides.yaml]: Done, thanks

**Review by dustymabe — APPROVED** [review]
LGTM

**Comment by dustymabe** [comment]
Note. Because of the current timing this won't land in `testing` for another two weeks. If you want it to land in `next` this week then I suggest you open a `next-devel` PR as well.

**Comment by cgwalters** [comment]
I guess I might as well just wait on the bodhi update to go stable :cry:

**Comment by jlebon** [comment]
rpm-ostree's CI (and almost everything else) uses testing-devel so once this merges, we should be unblocked.

File: manifest-lock.overrides.yaml [hunk 2]
```diff
@@ -8,4 +8,10 @@
 # in the `metadata.reason` key, though it's acceptable to omit a `reason`
 # for FCOS-specific packages (ignition, afterburn, etc.).
 
-packages: {}
+packages:
+  skopeo:
+    evr: 1.5.1-1.fc35
```
Comment (jlebon) [inline manifest-lock.overrides.yaml:13]: Ouch, looks like it was also missing an epoch:

```suggestion
    evr: 1:1.5.1-1.fc35
```

**Comment by dustymabe** [comment]
`skopeo-1.5.1-1.fc35` is [stable now](https://bodhi.fedoraproject.org/updates/FEDORA-2021-b8fee395ca) so this PR is obsolete.

**Pull request closed without merging**